### PR TITLE
Rename Object into ZuoraObject to support PHP 7.2

### PR DESCRIPTION
**Pull request by varvashenya**
- [x] Renamed Object into ZuoraObject in order to support PHP 7.2
- [x] updated phpunit to 7.5
- [x] minimum supported PHP version bumped to ^7.1
- [x] fixed php-coveralls
- [x] php matrix in travis now 7.1, 7.2 & 73
- [x] fix issues identified by `phpcs --standard=PSR2 ./src/`

### CODE REVIEW

**Comment by mhrabovcin** [comment]
Thanks for the PR @varvashenya. It seems that tests are failing on PHP `7.2` and `7.3` with error

```
Fatal error: Declaration of PHPUnit_Framework_Comparator_DOMDocument::assertEquals($expected, $actual, $delta = 0, $canonicalize = false, $ignoreCase = false) must be compatible with PHPUnit_Framework_Comparator_Object::assertEquals($expected, $actual, $delta = 0, $canonicalize = false, $ignoreCase = false, array &$processed = Array) in /home/travis/build/mhrabovcin/zuora-rest/vendor/phpunit/phpunit/PHPUnit/Framework/Comparator/DOMDocument.php on line 114
```

Maybe you have to bump also PHPUnit?

**Comment by varvashenya** [comment]
Yep. It's 1st item in my to-do list after lunch

File: test/Zuora/Test/Object/ZuoraObjectTest.php [hunk 1]
```diff
@@ -54,7 +55,7 @@ public function testdMapWrongArgumentType()
     public function testdMapWrongClassname()
     {
         $data = array('test' => array());
-        $object = new Object($data);
+        $object = new ZuoraObject($data);
         $object->map('test', 'MissingClass');
     }
 }
```
Comment (stmixnovich) [inline test/Zuora/Test/Object/ZuoraObjectTest.php]: EOF

**Review by stmixnovich — APPROVED** [review]

File: test/Zuora/Test/Object/ZuoraObjectTest.php (hunk 1 above)
Comment (varvashenya) [reply test/Zuora/Test/Object/ZuoraObjectTest.php]: Thanks! Addressed as part of `phpcs --standard=PSR2 ./test/`
Please re-visit

**Comment by mhrabovcin** [comment]
Thanks, @varvashenya the patch looks great. Maybe you can add `phpcs --standard=PSR2 ./src/` as part of travis run?

**Review by gcassie — APPROVED** [review]

**Review by stmixnovich — APPROVED** [review]

**Comment by mhrabovcin** [comment]
Thanks @varvashenya @gcassie @stmixnovich !

We can always add `phpcs` CI check in a separate PR. This is a great improvement for the library now being compatible with PHP 7.1-7.3.

**Pull request merged**